### PR TITLE
section 11 touch-up, retitled sect7-9 headings, reorg'd sect4-6

### PR DESCRIPTION
**Pull request by haochizzle**
Touching up section 11 
* Capitalization 
* Punctuation
* Grammar
* Messaging

Retitled headings for sections 7-9 to be more specific ("NFT")

Reorganized sections 4-6 to logically flow better (start with ERC-20 and end with ERC-1155)

Closes: [#134](https://github.com/ChainSafe/devrel/issues/134)

### CODE REVIEW

File: docs/v2/11_web3wallet-mobile-and-desktop.md [hunk 1]
```diff
@@ -172,39 +173,43 @@ public class Web3WalletSignVerify : MonoBehaviour
...
+Step 2: Clone the [Web3Wallet](https://github.com/ChainSafe/game-web3wallet) repo using GitHub Desktop. Press `Add`, `Clone Repository...`, and then paste in the following URL: `https://github.com/ChainSafe/game-web3wallet`.
 
 ![](<v2Assets/image (14).png>)
 
-Step 3: If it doesn't ask you automatically, in the top left, press add, then press clone repository and paste in the URL [https://github.com/ChainSafe/game-web3wallet](https://github.com/ChainSafe/game-web3wallet)
-
 ![](<v2Assets/image (12).png>)
 
-Step 4: GitHub desktop will ask you if you want to fork the repo. When prompted select "For My Own Purposes". This will create a fork that you can push to your own repository.
+Step 3: GitHub Desktop will ask you if you want to fork the repo. When prompted, select `For My Own Purposes`. This will create a fork, where you can push to your own repository.
```
Comment (waymobetta) [inline docs/v2/11_web3wallet-mobile-and-desktop.md:186]: This assumption is that the user is using Github Desktop?

File: docs/v2/11_web3wallet-mobile-and-desktop.md [hunk 2]
```diff
@@ -172,39 +173,43 @@ public class Web3WalletSignVerify : MonoBehaviour
...
 
-You can find the Web3Wallet repo here [https://github.com/ChainSafe/game-web3wallet](https://github.com/ChainSafe/game-web3wallet) you can download and host this on git for free. This instantiates web3 in webpage to allow MetaMask to connect & make transactions. We'll guide you through this process below.
+You can find the Web3Wallet repo [here](https://github.com/ChainSafe/game-web3wallet). You can download and host this on git for free. This instantiates web3 in a webpage to allow MetaMask to connect & make transactions. We'll guide you through this process below.
 
-Step 1: Create a GitHub account [https://github.com/join](https://github.com/join)\
+Step 1: [Create a GitHub account](https://github.com/join).
 
-
-Step 2: Clone the web3wallet repo using GitHub Desktop
+Step 2: Clone the [Web3Wallet](https://github.com/ChainSafe/game-web3wallet) repo using GitHub Desktop. Press `Add`, `Clone Repository...`, and then paste in the following URL: `https://github.com/ChainSafe/game-web3wallet`.
```
Comment (waymobetta) [inline docs/v2/11_web3wallet-mobile-and-desktop.md:180]: We might be able to get away with just saying "Clone the repo" and not need to point them to use Github Desktop; in fact, I think we don't want to advertise GD usage bc that is kind of potato compared to command line (and will poorly reflect) unless we have previously said somewhere that GD is to be used?

**Review by waymobetta — COMMENTED** [review]
Overall, changes look good. Though I would generally make the suggestion to remove all usage of Github Desktop across the documentation unless this was something specifically suggested by the Gaming team to have.

File: docs/v2/11_web3wallet-mobile-and-desktop.md (hunk 2 above)
Comment (haochizzle) [reply docs/v2/11_web3wallet-mobile-and-desktop.md:180]: Mention of GH Desktop is specifically what Sneakz had written in the first iteration of the v2 docs AFAICT. We can consider rewriting this in a refactor.

**Review by waymobetta — APPROVED** [review]

**Comment by sneakzttv** [comment]
branch just need to be updated as of today using the update branch button to get it on the same state as main
image

File: docs/v2/11_web3wallet-mobile-and-desktop.md (hunk 1 above)
Comment (haochizzle) [reply docs/v2/11_web3wallet-mobile-and-desktop.md:186]: We will have to rewrite this part in the refactor to be more inclusive, but this first pass-through is meant to be a touch-up and not anything structurally significant

**Pull request merged**